lib/controllers/ephemeral/account-linking/associated_accounts_controller.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:collection';
+import 'package:pispapp/models/consent.dart';
+
+import 'package:get/get.dart';
+
+class AssociatedAccountsController extends GetxController {
+  AssociatedAccountsController(this.associatedAccounts);
+  final List<Account> associatedAccounts;
+  final HashSet<String> _selectedAccIds = HashSet();
```

Comment:
Sorry but may I ask why don't we just use a set of `Account` instead of `String`?

Reply:
I think it's because Account doesn't implement equatable so the HashSet won't work with it's normal time efficiency.

Reply:
Maybe you can just implement the `hashCode` function for `Account` and returns the `hashCode` function of the id itself so we can have a hash set of Account here? Just thinking that it would be nicer to see and we can refer to the referenced object directly when accessing the hash set

Reply:
Ok